### ISSUE TITLE
All adjacent characters are highlighted on hover, not just the unicode escape

### ISSUE THREAD

**Issue by ChildishGiant** [jamesohortle/UnicodeHover#15]
![image](https://user-images.githubusercontent.com/13716824/105044465-ac5af580-5a5e-11eb-9e20-35d1fb4df22c.png)

**Comment by jamesohortle** [comment]
Yes, I'm unsure if this is something to do with UnicodeHover or the language server for JS that is tokenising the string. Feel free to investigate and submit a PR. The tool is otherwise working as expected so this is a low priority fix for me.
